manifests/l/Lapce/Lapce/0.1.1/Lapce.Lapce.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Lapce.Lapce
+PackageVersion: 0.1.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+ReleaseDate: 2022-05-19
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/lapce/lapce/releases/download/v0.1.1/Lapce-windows.msi
+  InstallerSha256: C804157D963033ACF5EA111A8E9C6F964320D708358E4112851164DF7BA72890
```

```suggestion
  InstallerSha256: 156496808fef8c6ca00edb0ebb9e969b80b10d20749d22c4ef6f92529ad8636f
```
Installer updated by @dzhou121